.spelling

```diff
@@ -480,6 +482,7 @@ reachability
 rearchitect
 recomposition
 redeployments
+relabeling
```

Comment:
hmm, is this really needed? What about `relabelling`?

Reply:
unfortunately, prom uses the single `l` variety.